python/lib/sift_py/upload/status.py

```diff
@@ -0,0 +1,61 @@
+import re
+import time
+from urllib.parse import urljoin
+
+import requests
+from sift_py.rest import SiftRestConfig
+
+
+class DataImportStatus:
+    STATUS_PATH = "/api/v1/data-imports"
+    _data_import_id: str
+
+    def __init__(self, restconf: SiftRestConfig, data_import_id: str):
+        base_uri = self.__class__._compute_uri(restconf)
+        self._data_import_id = data_import_id
+        self._status_uri = urljoin(base_uri, self.STATUS_PATH)
+        self._apikey = restconf["apikey"]
+
+    def get_status(self):
+        response = requests.get(
+            url=f"{self._status_uri}/{self._data_import_id}",
+            headers={"Authorization": f"Bearer {self._apikey}"},
+        )
+        response.raise_for_status()
+        return response.json().get("dataImport").get("status")
```

Comment:
As mentioned below, would be nice to convert this to a defined enum that has each member documented.